<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1659AS4" failures="1" tests="1" name="Surefire test" time="0.041" errors="0" timestamp="2023-11-09T07:49:42 IST">
  <testcase classname="com.example.CalculatorTest" name="testadd" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [6] but found [7]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [7]
at com.example.CalculatorTest.testadd(CalculatorTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testadd -->
</testsuite> <!-- Surefire test -->
